PlanningBiblio/PlanningBiblio
# MT29616 - Deny holidays if planning started or validated

This patchs adds the same behavior regarding started and validated plannings
    for holidays as it is currently implemented for absences.
    
    Two new configuration options have been added:
    
     - Conges-planningVide (same as Absences-planningVide but for holidays)
     - Conges-apresValidation (same as Absences-apresValidation but for holidays)
    
    Test plan:
    
     - Apply the patch
    
     - Check that the behavior for absences is the same as without the patch:
    
     - Uncheck Absences-planningVide and Absences-apresValidation
     - Try to add an absence on a started planning
     - Check that you can't (or can with confirmation if you are an admin)
     - Try to add an absence on a validated planning
     - Check that you can't (or can with confirmation if you are an admin)
    
     - Check that the behavior for holidays is the same as the one for absences:
    
     - Uncheck Conges-planningVide and Conges-apresValidation
     - Try to add an holiday on a started planning
     - Check that you can't (or can with confirmation if you are an admin)
     - Try to add an holiday on a validated planning
     - Check that you can't (or can with confirmation if you are an admin)
